### PR TITLE
Address NCEF feedback for NV

### PR DESCRIPTION
**Pull request by rpartridge**
NCEF emailed with some feedback -- see ticket with details: https://app.asana.com/0/1206629812394927/1207013718145444/f

Summary:
1. Links for NV Energy home energy saver program should go to the specific products, not the overall program link. To do this I split out the home energy saver program into several programs: air conditioning, water heating, weatherization ("home improvements"), and residential appliances. Those all link specifically to the appliance pages
2. The residential air conditioning program incentives were off, I'm guessing they were referencing 2023 numbers. This updates them with data from https://www.nvenergy.com/save-with-powershift/home-energy-saver/residential-ac-and-mid-stream/residential-air-conditioning
3. Updates the low income thresholds for the residential air conditioning program incentives. This wasn't mentioned in the feedback but I noticed when I was looking into this -- we were using the same low income thresholds for all NV energy incentives, but those thresholds should only apply to the [qualified appliance replacement program](https://www.nvenergy.com/save-with-powershift/qualified-appliance-replacement)

### CODE REVIEW

File: data/low_income_thresholds.json [hunk 1]
```diff
@@ -405,6 +402,24 @@
         "8": 115800
       }
     },
+    "nv-nv-energy-residential-air-conditioning": {
+      "incentives": [
+        "NV-23",
+        "NV-24",
+        "NV-25"
+      ],
+      "source_url": "https://dwss.nv.gov/Energy/3_Eligibility_Criteria/",
```
Comment (rpartridge) [inline data/low_income_thresholds.json:411]: This program doesn't specifically list these thresholds, but on their [website](https://www.nvenergy.com/save-with-powershift/home-energy-saver/residential-ac-and-mid-stream/residential-air-conditioning) they say: "Customers who currently participate in the Nevada Energy Assistance Program are automatically eligible to receive a higher incentive. Customers living in affordable housing properties listed under organizations such as Southern Nevada Regional Housing Authority (SNRHA), Reno Housing Authority (RHA), Nevada Rural Housing Authority (NRHA), Housing and Urban Development (HUD), and Section 8 Housing may also qualify for this increased incentive."

To simplify this, I found the thresholds for the Nevada Energy Assistance Program, which are listed here

Comment (oyamauchi) [reply data/low_income_thresholds.json:411]: I think there's an argument to be made for extending the `low_income` field somehow to express eligibility "by reference" to other programs -- i.e. if you're already eligible for LIHEAP or Section 8 or whatever, then you're eligible for this. But what you've done here is fine.

File: src/data/programs/nv_programs.ts [hunk 2]
```diff
@@ -1,10 +1,18 @@
 export const NV_PROGRAMS = {
-  nv_nVEnergy_homeEnergySaver: {
+  nv_nVEnergy_residentialAppliancesAndProducts: {
     name: {
-      en: 'Home Energy Saver',
+      en: 'Residential Appliances and Products',
```
Comment (oyamauchi) [inline src/data/programs/nv_programs.ts:4]: Could you put `NV Energy` at the start of the all the NV Energy program names? (See the last bullet of [this post](https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1711991416542819?thread_ts=1711991416.542819) for why)

File: data/NV/incentives.json [hunk 3]
```diff
@@ -482,21 +482,21 @@
...
+      "minimum": 1700,
+      "maximum": 3400
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "Up to $4,000 income qualified instant rebate from qualifying distributors when you replace old air condition system with an air source heat pump.",
-      "es": "Un reembolso instantáneo de hasta $4,000 dólares si cumple con el nivel de ingresos por distribuidores certificados para reemplazar su antiguo sistema de aire acondicionado por una bomba de calor con fuente de aire. "
+      "en": "Up to $3,400 income qualified instant rebate from qualifying distributors when you replace old air condition system with an air source heat pump.",
```
Comment (oyamauchi) [inline data/NV/incentives.json:496]: `air conditioning`

File: data/NV/incentives.json [hunk 4]
```diff
@@ -458,21 +458,21 @@
...
-      "en": "Up to $3,600 income qualified instant rebate when you replace your old air condition system with higher efficiency equipment.",
-      "es": "Un reembolso instantáneo de hasta $3,600 dólares que cumpla con el nivel de ingresos cuando sustituya su viejo sistema de aire acondicionado por un equipo de eficiencia superior. "
+      "en": "Up to $3,060 income qualified instant rebate when you replace your old air condition system with higher efficiency equipment.",
+      "es": "Un reembolso instantáneo de hasta $3,060 dólares que cumpla con el nivel de ingresos cuando sustituya su viejo sistema de aire acondicionado por un equipo de eficiencia superior. "
     },
-    "low_income": "nv-nv-energy"
+    "low_income": "nv-nv-energy-residential-air-conditioning"
   },
   {
     "id": "NV-24",
```
Comment (oyamauchi) [inline data/NV/incentives.json:478]: Should this have an exclusion relationship with NV-21? It looks like the income-qualified version of the same thing. Same with NV-25 and NV-22

**Review by oyamauchi — APPROVED** [review]
Approving, but there are some minor things to address

**Pull request merged**